### ISSUE TITLE
FRONTEND WITH FRAMEWORK: VueJS-2.4 => Filters in Vue.js

### ISSUE THREAD

**Issue by SourabhSaraswat-191939** [girlscript/winter-of-contributing#2309]
### Description

Contributors are required to explain about Filters in Vue.js, which should cover questions like:-
What is Filters in Vue.js?
When do we have to use this?
How does this work?
How to implement Filters with code (also give theory explanation of code)?
You can take 2-3 examples with codes of different types to explain this.
etc....

**Note:**

Plagiarism is strictly prohibited.
Changes should be made inside the Frontend_Web_Development_React_Angular_Vue/ dir inside Frontend_Web_Development_React_Angular_Vue branch.
Files need to be added into the Week2 folder only.

Domain

### Domain

Frontend Dev React/Angular/Vue

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

**Comment by SourabhSaraswat-191939** [comment]
If you don't know this framework. Don't worry still, you can contribute.
Here I am attaching some references for you.

https://vuejs.org/v2/guide/
https://www.tutorialspoint.com/vuejs/vuejs_overview.htm
https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks/Vue_getting_started

You can also study from any other source to learn.

Note -> These references contents are not to copy/paste. Avoid copy/paste, We will be checking.

**Comment by shivang02** [comment]
I would like to work on this issue. Please assign it to me.

**Comment by SourabhSaraswat-191939** [comment]
Hey @shivang02, any updates?
It's being too long, we are not getting any updates from your side.
If you are facing any problems while working on this issue. Let us know. We will help you out.

**Comment by shivang02** [comment]
Hi @SourabhSaraswat-191939 , really sorry I got busy with other tasks and delayed this but I'll finish it by Tuesday. I noticed that Filters have been deprecated so I am confused if I should explain it in details anyways.

**Comment by SourabhSaraswat-191939** [comment]
@shivang02  Yes,  it is deprecated in Vue3. But many of the developers use Vue2. So we have to take care of them also. Go with the Vue2 Filters concept.
And @shivang02, every issue after assigning to a contributor has 12 days of the time period to create a PR. Otherwise, the issue is reassigned to anyone else after 12 days. Please take care of these things.
No problem you can submit it by Tuesday. 

If you have any issues you can ping us on discord or Github.

**Comment by shivang02** [comment]
@SourabhSaraswat-191939 I created a pull request but I think I messed it up as it says it's to merge into the main branch while we had to merge it to the branch we were working on. Can you help me out?
